src/core/mergeDocument.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * https://www.pdftron.com/api/web/CoreControls.Document.html#mergeDocument__anchor
+ */
+import actions from 'actions';
+
+export default (dispatch, documentToMerge, position) => {
+  dispatch(actions.openElement('loadingModal'));
+  return window.readerControl.docViewer.getDocument().mergeDocument(documentToMerge, position)
```

Comment:
Hmm as I said we don't use readerControl internally. APIs on the readerControl are only for external usage. Let me know if you are not clear about what I mean.

Reply:
Thanks for pointing this out, I added the API file but forgot to get rid of the "readerControl", now this uses `window.docViewer`